src/main/js/dynamic/DistributionConfiguration.js

```diff
@@ -196,6 +196,7 @@ class DistributionConfiguration extends Component {
                     fieldErrors={this.props.fieldErrors}
                     self={this}
                     stateName="providerConfig"
+                    provider={currentProvider.name}
```

Comment:
why are we passing provider here? Is this left over from some previous old setup?